livekit-plugins/livekit-plugins-fal/livekit/plugins/fal/stt.py

```diff
@@ -0,0 +1,132 @@
+import dataclasses
+import io
+import os
+import wave
+from dataclasses import dataclass
+
+import fal_client
+from livekit.agents import stt
+from livekit.agents.stt import SpeechEventType, STTCapabilities
+from livekit.agents.utils import AudioBuffer, merge_frames
+
+from .log import logger
+
+
+@dataclass
+class _STTOptions:
+    language: str
+    task: str
+    chunk_level: str
+    version: str
+
+
+class WizperSTT(stt.STT):
+    def __init__(
+        self,
+        *,
+        language="en",
+        task="transcribe",
+        chunk_level="segment",
+        version="3",
+        api_key=None,
+    ):
+        super().__init__(
+            capabilities=STTCapabilities(streaming=False, interim_results=True)
+        )
+        self._api_key = api_key or os.getenv("FAL_API_KEY")
```

Comment:
fwiw it doesn't look like `self._api_key` is used anywhere, and the fal client seems to expect a different env var, `FAL_KEY`, so this isn't working out of the box for me right now. Hopefully helpful!

Reply:
good catch @egoldschmidt  I will update it next commit.